Side-effect of changing username on codeowners is declared (#23842)
github/docs: content/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/changing-your-github-username.md

After changing your username, the URLs to any public or secret gists will also change and previous links to these will return a 404 error. We recommend updating the links to these gists anywhere you may have shared them.

## CODEOWNERS files

> After changing your username, CODEOWNERS files will need to be manually updated. When viewing the CODEOWNERS files from the web, a warning will show if the file contains any unknown user(s) or user(s) without write access. We recommend updating all relevant CODEOWNERS files with your new username.

Reviewer: ```suggestion
After changing your username, CODEOWNERS files need to be manually updated. When viewing the CODEOWNERS files on the web, a warning displays if the file contains any unknown user(s) or user(s) without write access. We recommend updating all relevant CODEOWNERS files with your new username.
```

Reviewer: What does `the web` mean in this context, or where would a user view the CODEOWNERS file(s) online? If possible, be more specific. For example, if `the web` actually means GitHub.com, then let's say that.

Author: Hi @rachaelrenk, thanks for the review! The changes suggested were addressed in this [PR](https://github.com/github/docs/pull/23859) - which was created for another issue but happened to have the changes made here - and has now been merged in. So I will close this PR. Thank you!
